src/include/daos_srv/vos.h

```diff
@@ -81,6 +120,60 @@ vos_dtx_list_cos(daos_handle_t coh, daos_unit_oid_t *oid,
 int
 vos_dtx_list_committable(daos_handle_t coh, struct daos_tx_entry **dtes);
 
+/**
+ * Check whether the specified DTX can be committed or not.
+ *
+ * \param coh	[IN]	Container open handle.
+ * \param dti	[IN]	The DTX identifier.
+ *
+ * \return		See enum dtx_status for kinds of cases.
+ *			Negative value if error.
+ */
+int
+vos_dtx_check_committable(daos_handle_t coh, struct daos_tx_id *dti);
+
+/**
+ * Commit the specified DTXs.
+ *
+ * \param coh	[IN]	Container open handle.
+ * \param dtis	[IN]	The array for DTX identifiers to be committed.
+ * \param count [IN]	The count of DTXs to be committed.
+ *
+ * \return		Positive value if need to aggregate some old DTXs.
```

Comment:
again, does this mean you _need_ to aggregate them or that there are some available to aggregate and more of a hint that aggregation _can_ be done ?

Reply:
It is "suggest" to do DTX aggregation.